extension.js

```diff
@@ -158,13 +163,12 @@ function _showUI() {
   let monitor = Main.layoutManager.primaryMonitor;
   container.set_width(monitor.width);
   container.set_height(monitor.height);
-  container.set_position(monitor.x, monitor.y);
+  container.set_position(monitor.x, 200);
```

Comment:
Here the purpose of setting the container to take up the whole screen is that clicking anywhere on the container closes the switcher. The point there is that I want to make it as unlikely as possible that the user feels stuck with the switcher, as there is no visual indication of how to get rid of it (pressing Enter and Esc are two ways besides clicking anywhere, but it's best to play it safe with something that prevents any other use of the computer).

With this change clicking at the top of the screen doesn't do anything.